backend/api/EventHandlers/ScheduledMissionEventHandler.cs

```diff
@@ -72,14 +72,17 @@ private async Task<bool> StartScheduledMission(ScheduledMission scheduledMission
             {
                 var report = await _isarService.StartMission(robot: scheduledMission.Robot, echoMissionId: scheduledMission.EchoMissionId);
                 _logger.LogInformation("Started mission {id}", scheduledMission.Id);
+                await _scheduledMissionService.Delete(scheduledMission.Id);
```

Comment:
Should the scheduled mission be deleted upon completion instead? In case it fails, it might be nice to be able to reschedule it, using the same instance